app/models/event.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class Event < ApplicationRecord
+  belongs_to :creator, class_name: 'User', foreign_key: 'user_id'
+
+  has_many :user_attended_events
+  has_many :attendees, through: :user_attended_events, source: :user
+
+  has_many :invitations
+  has_many :invitees, through: :invitations, source: :user
+
+  scope :past, -> { where('date < ?', Time.now) }
+  scope :upcoming, -> { !past }
```

Comment:
The logic here is not correct.
Both past events and upcoming events are displayed as upcoming events.
See this :point_down: 
![Screenshot from 2020-02-19 01-49-54](https://user-images.githubusercontent.com/44072711/74809034-45671680-52ba-11ea-8539-3e2535726af9.png)
Let's fix this by using `scope :past, -> { where('date > ?', Time.now) }`

Reply:
Thanks for noticing this. We fixed the bug, but with a different method